ui-components/src/locales/vi.json

```diff
@@ -729,6 +729,7 @@
     "unitType": "Loại Căn nhà",
     "viewMap": "Xem Bản đồ",
     "viewOnMap": "Xem trên Bản đồ",
+    "website": "Trang mạng",

```

Comment:
we've been letting our vi and zh json files go out of date with our general.json since there's no plans to reconcile the general.json with upstream until Exygy has a better jurisdiction string strategy.

Reply:
Ah I see. Is it important to revert my changes to vi and zh? (it just feels strange to intentionally force them to be out of sync)